Automattic/wp-calypso
# New ThemeCard component

Related to #75457

## Proposed Changes

This PR creates the component `<ThemeCard />`, and updates the Theme Showcase to use this component to render theme cards. For this PR, `<ThemeCard />` still depend of several components to be passed as props (e.g.: banner, badge, or options). We can work on moving those components to be rendered within `<ThemeCard />` in future PRs. 

![Screenshot 2023-04-10 at 12 14 06 PM](https://user-images.githubusercontent.com/797888/230842443-f6abe3e7-96ca-4425-a645-c22b51b39847.png)

After this PR, it would be good to remove unused CSS code as a result of this change.

## Testing Instructions



* Head to the Theme Showcase.
* Ensure that the theme card UI and its functionality works the same as the one in production.
* Please also test the logged-out Theme Showcase.
* And also test Atomic sites.

## Pre-merge Checklist
